api/app/Http/Controllers/Api/AuthController.php

```diff
@@ -77,4 +81,75 @@ public function register(UserRegistrationModel $request)
 
         User::create($account);
     }
+
+    /**
+     * Send a mail for resetting the password.
+     *
+     * @param \App\Http\Requests\Api\PasswordResetModel $request
+     *
+     * @return mixed
+     */
+    public function sendResetMail(PasswordResetModel $request)
+    {
+        $user_email = $request->email;
+        $user = User::where('email', $user_email)->first();
+        if ($user && $this->notSpamming($user, env('PASSWORD_RESET_MINUTES', 30))) {
+            // User exists and had no request < PASSWORD_RESET_MINUTES
+            $uuid_token = Uuid::generate(4);
```

Comment:
Using uuid itself is secure enough since it's only available for a limited amount of time. Besides that the generated token is long enough and consists of alphanumerical characters. The chance the bruteforcing will success is nothing compared to guessing the password of the user itself. Using str_random and uuid combined would be wasted energy. But indeed, I agree, "more secure".